src/zcash/address/bip44.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2021 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+#include "bip44.h"
+
+std::optional<std::pair<CExtKey, HDKeyPath>> libzcash::DeriveBip44TransparentAccountKey(const HDSeed& seed, uint32_t bip44CoinType, libzcash::AccountId accountId) {
```

Comment:
No, this was an intentional change: `HDSeed` is in zip32.h, and the only capability required for this method is obtaining the fingerprint for the seed. This is actually important, because `MnemonicSeed::Random` needs to be able to depend upon `unified.h` without creating a cycle.

Reply:
In that case, either make this change in a commit prior to the move-only commit, or explicitly call out this change in the commit message so that it isn't assumed to be move-only / making no functional changes.